pkgs/applications/networking/instant-messengers/discord-screenaudio/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, wrapQtAppsHook
+, pkg-config
+, qtbase
+, qtwebengine
+, knotifications
+, kxmlgui
+, kglobalaccel
+, pipewire
+, xdg-desktop-portal
+}:
+
+stdenv.mkDerivation rec {
+  pname = "discord-screenaudio";
+  version = "1.4.3";
+
+  src = fetchFromGitHub {
+    owner = "maltejur";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "QzKvT0JibTDm5y/KgB4IPWU3L42eznGi97e0PEcSL3k=";
+    fetchSubmodules = true;
+  };
+
+  NIX_CFLAGS_COMPILE = [
+    "-I${pipewire.dev}/include/pipewire-0.3"
+    "-I${pipewire.dev}/include/spa-0.2"
+    "-Wno-pedantic" # Fails without flag
+  ];
```

Comment:
I don't know if this is the best way to do this, but the build fails without it.